.circleci/config.yml

```diff
@@ -49,10 +49,11 @@ jobs:
             - node_modules
```

Comment:
what this for ?

Reply:
i don't know this is template from circleci